### PR TITLE
[WIP] Add Japanese translation(basic)

### LINKED ISSUE #94: Translate into Japanese
We'll translate the parcel website into Japanese!

### PR DESCRIPTION
**Pull request by Pittan**
Fixes #94 .

I translated basic articles into Japanese 🙌 
Comments from Japanese speaking users are welcome, especially on technical terms.

### CODE REVIEW

**Comment by fathyb** [comment]
Awesome! I added `[WIP]` in your title so mergers know it's in progress. Feel free to remove it when ready 🙂

**Comment by imncls** [comment]
I will review it.
Please give me a little time.

**Comment by Pittan** [comment]
@fathyb Reviewer account is gone :cry:

**Comment by imbsky** [comment]
I changed account.
I’m working it now.

File: src/i18n/ja/docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  <script src="./index.js"></script>
+</body>
+</html>
+```
+
+```javascript
+console.log("hello world");
+```
+
+Parcelは開発用サーバを内蔵しています。それはあなたがファイルを変更すると自動で再ビルドを行い、素早い開発のために[ホットモジュールリプレースメント](hmr.html)をサポートしています。
```
Comment (imbsky) [inline src/i18n/ja/docs/getting_started.md]: Please delete whitespaces.

File: src/i18n/ja/docs/production.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# ✨ 本番
```
Comment (imbsky) [inline src/i18n/ja/docs/production.md]: > `# ✨ Production`

```diff
-# ✨ 本番
+# ✨ プロダクション
```

File: src/i18n/ja/docs/production.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+本番用にアプリケーションをバンドルするときは、Parcelのプロダクションモードを使用できます。
```
Comment (imbsky) [inline src/i18n/ja/docs/production.md]: > When it comes time to bundle your application for production, you can use Parcel's production mode.

```diff
-本番用にアプリケーションをバンドルするときは、Parcelのプロダクションモードを使用できます。
+プロダクション向けにアプリケーションをバンドルするときは、Parcelのプロダクションモードを使用できます。
```

File: src/i18n/ja/index.html [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        <h3>✂️ 設定なしのコード分割</h3>
+        <p>動的な <code>import()</code> 記法で、Parcelは出力結果を分割します。最初の読み込みに必要なファイルだけをバンドルすることができます。</p>
+      </section>
+      <section>
+        <h3>🔥 ホット モジュール リプレースメント</h3>
+        <p>あなたが開発中に変更を加えると、Parcelは自動的にブラウザに読み込まれたモジュールを更新します。設定は必要ありません。</p>
+      </section>
+      <section>
+        <h3>🚨 親切なエラーロギング</h3>
+        <p>Parcelはエラーが発生した場合に、あなたが読みやすいシンタックスハイライトされたコードを出力します。原因の特定を助けます。</p>
```
Comment (imbsky) [inline src/i18n/ja/index.html]: > Parcel prints syntax highlighted code frames when it encounters errors to help you pinpoint the problem.

```diff
-Parcelはエラーが発生した場合に、あなたが読みやすいシンタックスハイライトされたコードを出力します。原因の特定を助けます。
+Parcelはエラーが発生した場合、シンタックスハイライトされたコードを出力する事で問題を特定しやすくします。
```

File: src/i18n/ja/index.html [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+      <section>
+        <h3>✂️ 設定なしのコード分割</h3>
+        <p>動的な <code>import()</code> 記法で、Parcelは出力結果を分割します。最初の読み込みに必要なファイルだけをバンドルすることができます。</p>
+      </section>
+      <section>
+        <h3>🔥 ホット モジュール リプレースメント</h3>
+        <p>あなたが開発中に変更を加えると、Parcelは自動的にブラウザに読み込まれたモジュールを更新します。設定は必要ありません。</p>
+      </section>
+      <section>
+        <h3>🚨 親切なエラーロギング</h3>
```
Comment (imbsky) [inline src/i18n/ja/index.html]: > 🚨 Friendly error logging

```diff
-🚨 親切なエラーロギング
+🚨 分かりやすいエラーログ
```

**Comment by imbsky** [comment]
This review has not been completed yet.
But I'm busy.
Maybe I don't have much time to use for this pull request this month.
So I submit this temporarily.
Sorry for late.
By the way, You don't need to change all.
Because my English is not perfect.

File: src/i18n/ja/layout/page.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+              <option class="zh" value="https://zh.parceljs.org">🇨🇳 中文</option>
+            </select>
+          </form>
+      <a href="https://github.com/parcel-bundler/parcel" target="_blank">GitHub</a>
+    </div>
+  </header>
+  <div class="content">
+    <nav>
+      <ul>
+        <li><a href="getting_started.html">🚀 はじめに</a></li>
```
[the remark this answers is not in the document]
Comment (Pittan) [reply src/i18n/ja/layout/page.html]: （日本語で失礼します）最初に見てほしいようなところなので、この場合は「はじめに」が良いかと思います。https://ejje.weblio.jp/content/Getting+Started

**Comment by Pittan** [comment]
Thank you for reviewing, and I'm sorry to be late :bow:  
All fixed 🏁

File: src/i18n/ja/layout/page.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (imbsky) [reply src/i18n/ja/layout/page.html]: @Pittan そういうことなら良いと思います！

**Comment by imbsky** [comment]
No worries.
I guess we should review all again.
But, I don't have much time.
Can anyone do that?

File: src/i18n/ja/index.html [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+    gtag('config', 'UA-110647385-1');
+  </script>
+</head>
+<body>
+  <header>
+    <h1>
+        <img class="logo" src="assets/logo.svg" alt="Parcel">
+      </h1>
+      <p class="subtitle">Blazing fast, zero configuration web application bundler</p>
+
```
[the remark this answers is not in the document]
Comment (imbsky) [reply src/i18n/ja/index.html]: @Pittan Why didn't you delete these?

File: src/i18n/ja/index.html [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+        <div>
+          <ul>
+            <li><a href="getting_started.html">🚀 はじめに</a></li>
+            <li><a href="assets.html">📦 アセット</a></li>
+            <li><a href="transforms.html">🐠 変換</a></li>
+          </ul>
+          <ul>
+            <li><a href="code_splitting.html">✂️ コード分割</a></li>
+            <li><a href="hmr.html">🔥 ホットモジュールリプレースメント</a></li>
+            <li><a href="production.html">✨ 本番</a></li>
```
[the remark this answers is not in the document]
Comment (imbsky) [reply src/i18n/ja/index.html]: @Pittan Change this

**Comment by imbsky** [comment]
Well, this is still WIP.
We don't have to hurry. (Maybe)

**Comment by imbsky** [comment]
Close or Add more?

**Comment by imbsky** [comment]
@Pittan Please close or continue.

**Pull request closed without merging**